2021_acm_recsys_transformers4rec/README.md

```diff
@@ -13,3 +13,8 @@ In order to demonstrate the usefulness of the library for research and also to v
 - [Appendix A - Techniques used in Transformers4Rec Meta-Architecture](Appendices/Appendix_A-Techniques_used_in_Transformers4Rec_Meta-Architecture.md)
 - [Appendix B - Preprocessing and Feature Engineering](Appendices/Appendix_B-Preprocessing_and_Feature_Engineering.md)
 - [Appendix C - Hypertuning - Search space and best hyperparameters](Appendices/Appendix_C-Hyperparameters.md)
+
+## Experiments reproducibility
+The experiments for the Transformers4Rec paper were perfomed in a former version of the Transformers4Rec library tagged as [recsys2021](https://github.com/NVIDIA-Merlin/Transformers4Rec/tree/recsys2021), which can be used for paper experiments reproducibility. We provide scripts for preprocessing the datasets [here](https://github.com/NVIDIA-Merlin/Transformers4Rec/tree/recsys2021/datasets), i.e., for creating features and grouping interactions features by sessions. But for your convenience we also provide the [pre-processed version of the datasets](https://drive.google.com/drive/folders/1fxZozQuwd4fieoD0lmcD3mQ2Siu62ilD?usp=sharing) for download, so that you jump directly into running experiments with Transformers4Rec. There is an example on how to setup your environment and run the training and evaluation scripts [here](https://github.com/NVIDIA-Merlin/Transformers4Rec/tree/recsys2021/hf4rec) and the best hyperparameters found for each experiment group (dataset and algorithm) can be found in our [Appendix C](Appendices/Appendix_C-Hyperparameters.md).  
+
+It is worthwhile to mention that since our experiments for the paper were finished the library was completely refactored by NVIDIA Merlin, to convert it from a research library into an flexible [open-source library](https://github.com/NVIDIA-Merlin/Transformers4Rec), packaged with PyTorch and Tensorflow APIs. We are working on preparing Python scripts to reproduce our paper results with the refactored version of Transformers4Rec and will release those scripts soon in this [NVIDIA-Merlin/publications](https://github.com/NVIDIA-Merlin/publications/tree/main/2021_acm_recsys_transformers4rec) repo, so that you can more easily understand and extend the library for your own research on sequential and session-based recommendation ;)
```

Comment:
```suggestion
It is worthwhile to mention that since our experiments for the paper were finished the library was completely refactored by NVIDIA Merlin, to convert it from a research library into an flexible [open-source library](https://github.com/NVIDIA-Merlin/Transformers4Rec), packaged with PyTorch and TensorFlow APIs. We are working on preparing Python scripts to reproduce our paper results with the refactored version of Transformers4Rec and will release those scripts soon in this [NVIDIA-Merlin/publications](https://github.com/NVIDIA-Merlin/publications/tree/main/2021_acm_recsys_transformers4rec) repo, so that you can more easily understand and extend the library for your own research on sequential and session-based recommendation ;)
```